py/picca/fitter2/parser.py

```diff
@@ -75,15 +82,17 @@ def parse_data(filename):
             value = float(value)
         if item == "ell-max":
             value = int(value)
-        if item == 'zref':
-            value = float(value)
         dic_init['data'][item] = value
 
     dic_init['cuts'] = {}
     for item, value in cp.items('cuts'):
         dic_init['cuts'][item] = float(value)
 
     dic_init['model'] = {}
+    dic_init['model']['zeff'] = zeff
+    dic_init['model']['zref'] = fiducial['zref']
+    dic_init['model']['Om']   = fiducial['Om']
+    dic_init['model']['OL']   = fiducial['OL']
```

Comment:
I'm surprised that coadacy doesn't complaint about these extra spaces. Could you please remove the double spaces here?

Reply:
done